GraphQL-TSC.md

```diff
@@ -0,0 +1,253 @@
+
+### Meetings
+
+In order to hold a GraphQL meeting for the TSC, a project, or a working group, you must do the following:
+
+* **Post an agenda** on GitHub before the meeting. For an example, please see the [GraphQL WG agendas](https://github.com/graphql/graphql-wg/tree/master/agendas).
+* **Record attendance** by ensuring every attendee PRs themselves onto the meeting agenda. This ensures that everybody has signed the right membership agreements. This process is free and fast, and is kicked off automatically when you open your first PR.
+* **Re-verify** at the start of each meeting that all participants have signed the membership document. If they haven't they can either add themselves to the meeting agenda and complete the forms, or disconnect and listen to the replay.
+* **Announce** at the beginning of the call that it is being recorded.
+* **Keep minutes** of the discussion (the recording can help with this). For an exmaple, please see the [GraphQL WG minutes](https://github.com/graphql/graphql-wg/tree/master/notes).
+
+Our Zoom account supports recording and livestreaming. We strongly encourage you to use the cloud recording feature. Staff can then download and post the videos to the [GraphQL YouTube channel](https://youtube.graphql.org). Alternately, if you want to livestream a meeting directly to YouTube, you can request access by emailing [operations@graphql.org](mailto:operations@graphql.org).
+
+If you have questions about these processes, please contact [operations@graphql.org](mailto:operations@graphql.org) and we can help provide context for why they are in place.
+
+## TSC Members
+
+### Who can participate on the TSC
+
+TSC meeting are open to anybody who wishes to participate. Aside from items which require a formal vote (such as approving specification verions or elections), anyone who has signed a Specification Membership Agreement may participate in discussions and decisions. As defined in the charter, the TSC will use the consensus decision-making process as often as possible.
+
+The TSC will also include a number of individuals who have been elected as voting members. These individuals are expected to be active participants on the TSC, and to be engaged and responsive on matters that require a formal vote.
+
+You must add yourself to the meeting agenda via PR in order to be able to join a GraphQL specification meeting.
+
+### TSC Voting Members
+
+The current voting members of the GraphQL TSC are:
+
+| Name | GitHub | Affiliation | Term begins  | Term ends |
+| --- | --- | --- | --- | --- |
+| Lee Byron (chair) |[leebyron](https://github.com/leebyron) | GraphQL Specification | N/A | N/A |
+| Andi Marek |[andimarek](https://github.com/andimarek) | Company  | TBD | TBD |
```

Comment:
@brianwarner My company affiliation is "GraphQL Java". thanks

Reply:
You told me this and I forgot to make the change, I'm sorry. I've just pushed an update.